pokemon-league-participation-manager/src/main/java/cz/muni/fi/pa165/pokemon/league/participation/manager/entities/PokemonSpecies.java

```diff
@@ -0,0 +1,99 @@
+package cz.muni.fi.pa165.pokemon.league.participation.manager.entities;
+
+import cz.muni.fi.pa165.pokemon.league.participation.manager.enums.PokemonType;
+import java.util.Objects;
+
+/**
+ * Entity class for object PokemonSpecies.
+ * 
+ * @author Jiří Medveď 38451
+ */
+public class PokemonSpecies {
+    
+    private Long id;
+    private String specieName;

```

Comment:
Not sure, but I don't think "specie" is a proper form, and google translate seems to agree - specie is something to do with coins. Just use speciesName

Reply:
renamed to speciesName